A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -3419,6 +3419,25 @@
             </UseTemplate>
         </Component>
 
+        <Component ID="Warning">
+            <UseTemplate Name="FBW_Push_Held">
+                <NODE_ID>PUSH_AUTOPILOT_MASTERAWARN_L</NODE_ID>
+                <HOLD_SIMVAR>L:PUSH_AUTOPILOT_MASTERAWARN_L</HOLD_SIMVAR>
+            </UseTemplate>
+            <UseTemplate Name="FBW_Push_Held">
+                <NODE_ID>PUSH_AUTOPILOT_MASTERAWARN_R</NODE_ID>
+                <HOLD_SIMVAR>L:PUSH_AUTOPILOT_MASTERAWARN_R</HOLD_SIMVAR>
+            </UseTemplate>
+            <UseTemplate Name="FBW_Push_Held">
+                <NODE_ID>PUSH_AUTOPILOT_MASTERCAUT_L</NODE_ID>
+                <HOLD_SIMVAR>L:PUSH_AUTOPILOT_MASTERCAUT_L</HOLD_SIMVAR>
+            </UseTemplate>
+            <UseTemplate Name="FBW_Push_Held">
+                <NODE_ID>PUSH_AUTOPILOT_MASTERCAUT_R</NODE_ID>
+                <HOLD_SIMVAR>L:PUSH_AUTOPILOT_MASTERCAUT_R</HOLD_SIMVAR>
+            </UseTemplate>
+        </Component>
```

Comment:
Adding this broke the ability to press the MASTER WARN and MASTER CAUT button to stop sounds. E.g. when you press the throttle fully forward with the parking brake set, you cannot stop the audio by pressing the MASTER WARN button anymore. The NODE_IDs are already used above. They're already assigned.

Reply:
I want to add that function on FWC because some warning should not canceled by master warning and caution.